backend/app/api/endpoints/admin/public_teams.py

```
@@ -0,0 +1,285 @@
+        team_json=team.json,
+        is_active=team.is_active,
+        created_at=team.created_at,
+        updated_at=team.updated_at,
+    )
+
+
+@router.get("/public-teams", response_model=PublicTeamListResponse)
+async def list_public_teams(
+    page: int = Query(1, ge=1),
+    limit: int = Query(20, ge=1, le=100),
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)